net/nginx/Makefile

```diff
@@ -67,181 +67,235 @@ PKG_CONFIG_DEPENDS := \
 
 include $(INCLUDE_DIR)/package.mk
 
-define Package/nginx
+define Package/nginx/default
   SECTION:=net
   CATEGORY:=Network
   SUBMENU:=Web Servers/Proxies
   TITLE:=Nginx web server
   URL:=http://nginx.org/
-  DEPENDS:=+NGINX_PCRE:libpcre +(NGINX_SSL||NGINX_HTTP_CACHE||NGINX_HTTP_AUTH_BASIC):libopenssl +NGINX_HTTP_GZIP:zlib +NGINX_LUA:liblua +libpthread
+  DEPENDS:=+NGINX_PCRE:libpcre +(NGINX_SSL||NGINX_HTTP_CACHE||NGINX_HTTP_AUTH_BASIC):libopenssl \
+     +NGINX_HTTP_GZIP:zlib +NGINX_LUA:liblua +libpthread
 endef
 
 define Package/nginx/description
  nginx is an HTTP and reverse proxy server, as well as a mail proxy server,
  written by Igor Sysoev.
 endef
 
+define Package/nginx
+  $(Package/nginx/default)
+  VARIANT:=no-ssl
```

Comment:
iirc, if you want PROVIDES and DEPENDS to work properly, you can't have a variant with the name as the same virtual package

Reply:
what you mean with a variant with the name as the same virtual package

Can you give me an example?